components/TextFields/src/MDCTextInputControllerOutlined.m

```diff
@@ -144,7 +144,7 @@ - (CGFloat)trailingViewTrailingPaddingConstant {
 // clang-format on
 - (UIEdgeInsets)textInsets:(UIEdgeInsets)defaultInsets {
   UIEdgeInsets textInsets = [super textInsets:defaultInsets];
-  CGFloat textVerticalOffset = self.textInput.placeholderLabel.font.lineHeight * .5f;
+  CGFloat textVerticalOffset = self.textInput.placeholderLabel.font.lineHeight * (CGFloat).5;
```

Comment:
tip: for consistency, let's make it `(CGFloat)0.5`

Reply:
Done throughout.